protocol/x/listing/types/msg_upgrade_isolated_perpetual_to_cross.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	fmt "fmt"
+
+	errorsmod "cosmossdk.io/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ sdk.Msg = &MsgUpgradeIsolatedPerpetualToCross{}
+
+func (msg *MsgUpgradeIsolatedPerpetualToCross) GetSigners() []sdk.AccAddress {
+	addr, _ := sdk.AccAddressFromBech32(msg.Authority)
+	return []sdk.AccAddress{addr}
+}
+
+func (msg *MsgUpgradeIsolatedPerpetualToCross) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Authority); err != nil {
```

Comment:
We don't need this check as you'll be checking the authority anyway so I think we can just get rid of this whole funciton